apps/css-workshop/pages/tree.html

```diff
@@ -1239,6 +1240,334 @@ <h2>With Icons</h2>
       </ul>
     </section>
 
+    <hr />
+    <h2>Small size</h2>
+    <section id="demo-sizes">
+      <ul
+        class="iui-tree"
+        role="tree"
+        data-iui-size="small"
+      >
+        <li
+          class="iui-tree-item"
+          role="treeitem"
+          tabindex="0"
+          aria-expanded="true"
+        >
+          <div class="iui-tree-node">
+            <input
+              class="iui-checkbox iui-checkbox-visibility iui-tree-node-checkbox"
+              type="checkbox"
+              checked
+            />
+            <div class="iui-tree-node-content">
+              <button
+                class="iui-button"
+                data-iui-variant="borderless"
+                data-iui-size="small"
+                type="button"
+              >
+                <svg-chevron-right-small
+                  class="iui-button-icon iui-tree-node-content-expander-icon  iui-tree-node-content-expander-icon-expanded"
+                  aria-hidden="true"
+                ></svg-chevron-right-small>
+              </button>
+              <svg-placeholder
+                class="iui-tree-node-content-icon"
+                aria-hidden="true"
+              ></svg-placeholder>
+              <span class="iui-tree-node-content-label">
+                <div class="iui-tree-node-content-title">Small tree</div>
```

Comment:
nit: Since now there is just a small tree demo, this can be reverted back to Facility 1.

Reply:
i'll leave this as-is since it's not part of the component code